src/Illuminate/Mail/Mailer.php

```diff
@@ -588,4 +591,20 @@ public function setQueue(QueueContract $queue)
 
         return $this;
     }
+
+    /**
+     * Dispatch the message processing event.
+     *
+     * @param  \Illuminate\Mail\Message  $message
+     * @param  array  $data
+     * @return void
+     */
+    protected function dispatchProcessingEvent($message, $data = [])
+    {
+        if ($this->events) {
+            $this->events->dispatch(
+                new MessageProcessing($message->getSwiftMessage(), $data)
```

Comment:
Why not Pass `\Illuminate\Mail\Message` to allow for using laravel api ?
And if the developper wants to swift message instance he can get him self.

Reply:
To be consistent with `MessageSending` and `MessageSent` events.